etna/whatson/models.py

```diff
@@ -589,3 +630,30 @@ def save(self, *args, **kwargs):
         "whatson.WhatsOnPage",
     ]
     subpage_types = []
+
+
+class EventFilterForm(forms.Form):
+    date = forms.DateField(
+        label="Choose a date",
+        required=False,
+        widget=forms.DateInput(attrs={"type": "date", "class": "filters__date"}),
+    )
+
+    category = forms.ModelChoiceField(
```

Comment:
Suggestion: name this `event_type` so it's there's an association with the type of the available options.